src/C-interface/dense/bml_allocate_dense_typed.c

```diff
@@ -112,16 +112,30 @@ bml_matrix_dense_t *TYPED_FUNC(
     bml_matrix_dimension_t matrix_dimension = { N, N, M };
     bml_matrix_dense_t *A =
         TYPED_FUNC(bml_zero_matrix_dense) (matrix_dimension, distrib_mode);
+#ifdef BML_USE_MAGMA
+    MAGMA_T *A_dense = calloc(N * N, sizeof(REAL_T));
```

Comment:
```suggestion
    MAGMA_T *A_dense = bml_allocate_memory(N * N, sizeof(REAL_T));
```
So that you get aligned allocation for supported compilers.

Reply:
done